consensus/src/pipeline/pipeline_builder.rs

```diff
@@ -192,8 +192,9 @@ impl PipelineBuilder {
         commit_proof: LedgerInfoWithSignatures,
     ) -> PipelineFutures {
         let prepare_fut = spawn_ready_fut(Arc::new(vec![]));
-        let execute_fut = spawn_ready_fut(());
-        let ledger_update_fut = spawn_ready_fut((compute_result.clone(), None));
+        let execute_fut = spawn_ready_fut(Duration::from_millis(0));
```

Comment:
Why not put timer in the tracker struct?

Reply:
I need to propagate the duration to the outside, it's used for backpressure, but I'll add some internal counters too